integration-tests/mongodb-panache/src/test/java/io/quarkus/it/mongodb/panache/reactive/ReactiveMongodbPanacheResourceTest.java

```diff
@@ -203,7 +203,7 @@ private void callReactiveBookEndpoint(String endpoint) throws InterruptedExcepti
                 nbEvent.increment();
             });
             source.open();
-            Thread.sleep(100);//wait a little for the events to comes in
+            Thread.sleep(150);//wait a little for the events to comes in
```

Comment:
Can't we use awaitility instead of this? We would wait for longer but still be more reactive as it would poll the result automatically.